Makefile

```diff
@@ -1712,8 +1719,8 @@ rustup-install-target-toolchain: rustup-set-version
 # BASE_BRANCH and BASE_TAG will be automatically determined if not specified.
 # See ./build.assets/changelog.sh
 .PHONY: changelog
-changelog:
-	@BASE_BRANCH=$(BASE_BRANCH) BASE_TAG=$(BASE_TAG) ./build.assets/changelog.sh
+changelog: $(CHANGELOG)
+	@$(CHANGELOG) --base-branch=$(BASE_BRANCH) --base-tag=$(BASE_TAG) ./
```

Comment:
No need to make this silent with `@`. If people don't like seeing the commands, they can run `make -s`. I like to know what commands are running

Reply:
Yup no problems changed it. I also like seeing the commands run helps to debug.